unittests/tools/test_mobsf_scorecard_parser.py

```diff
@@ -0,0 +1,108 @@
+from dojo.models import Engagement, Product, Test
+from dojo.tools.mobsf_scorecard.parser import MobSFScorecardParser
+from unittests.dojo_test_case import DojoTestCase
+
+
+class TestMobSFScorecardParser(DojoTestCase):
+
+    def test_parse_android_empty_file(self):
+
+        test = Test()
+        engagement = Engagement()
+        engagement.product = Product()
+        test.engagement = engagement
```

Comment:
This block in each function is not necessary

Reply:
Removed )